File: iOS7/iOS Learning 1/ios7 code/cha12/CrazyDrag/CrazyDrag/AboutViewController.m
//
//  AboutViewController.m
//  CrazyDrag
//
//  Created by happybubsy on 9/21/13.
//  Copyright (c) 2013 ioslearning. All rights reserved.
//

#import "AboutViewController.h"

@interface AboutViewController ()
- (IBAction)close:(id)sender;

@end

@implementation AboutViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)close:(id)sender {
    
    [self.presentingViewController dismissViewControllerAnimated:YES completion:nil];
}

@end

File: iOS11/Projects/cha16/BullsEye/BullsEye/AboutViewController.swift
//
//  AboutViewController.swift
//  BullsEye
//
//  Created by eseedo on 10/19/17.
//  Copyright © 2017 eseedo. All rights reserved.
//

import UIKit

class AboutViewController: UIViewController {
    
    @IBOutlet weak var webView: UIWebView!

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        
//        if let url = Bundle.main.url(forResource: "BullsEye",
//                                     withExtension: "html"){
//            if let htmlData = try? Data(contentsOf: url){
//                let baseURL = URL(fileURLWithPath: Bundle.main.bundlePath)
//                webView.load(htmlData, mimeType: "text/html",
//                             textEncodingName: "UTF-8",
//                             baseURL: baseURL)
//            }
//        }


        let myURL = URL.init(string: "http://www.apple.com")
        let request: URLRequest = URLRequest(url: myURL!)
        webView.loadRequest(request)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    @IBAction func close(){
        dismiss(animated:  true,completion: nil)
    }


}
